src/python/pants/engine/internals/platform_rules_test.py

```diff
@@ -178,7 +178,7 @@ def mock_environment_vars_subset(_req: EnvironmentVarsRequest) -> EnvironmentVar
         mock_gets=[
             MockGet(
                 output_type=EnvironmentVars,
-                input_types=(CompleteEnvironmentVars, EnvironmentVarsRequest),
```

Comment:
Fails to match the input types of the get it's trying to mock. It worked only because the check was broken, and in practice both the mock and the Get it satisfies do not take the `CompleteEnvironmentVars` arg.